LOCAL_DEVELOPMENT.md

```diff
@@ -13,29 +13,24 @@ Running the Lilypad application locally depends on the `.local.dev` file for sec
 A minimal local Lilypad network consists of the following pieces of infrastructure:
 
 - One blockchain node
-- One solver service
-- One job creator service
 - One IPFS node
 - One bacalhau node
+- One postgres database
+- One solver service
+- One job creator service
 - One resource provider service
 
 Order matters because the `solver`, `job creator` and `resource provider` will right away try to connect to the `blockchain node`. First, the `solver` will update the state of a known smart contract to publish a URL where other services can connect to it. Then, the `job creator`, and `resource provider` will fetch from the `blockchain` the URL for the `solver` and try to connect to it.
 
 ## Minimal local setup
 
-### 1. Blockchain node
+### 1. Base services
 
-The node can run directly from an existing docker image. It will initialize itself to a blank state with the admin address holding the funds to deploy the smart contracts (including the Lilypad token) and fund the accounts used by the different services. The blockchain is ephemeral, meaning every time you restart the node the state will be reset, you can work around this by keeping the node active if needed.
-
-These are the commands to run the node and boot the network: `./stack chain-clean` (the first time this won't do anything, but I find it better to get in the habit of resetting artifacts every time), `./stack chain` to run the node and `./stack chain-boot` to fund the accounts with ETH (for gas fees), compile the contracts, add Golang bindings to use the contracts directly in go code, deploy the contracts and fund the accounts with Lilypad tokens.
-
-#### Summary of command sequence
+The first step is to start the 3rd party services. This consists of a blockchain node (nitro), an IPFS node (kubo), a postgres database and a bacalhau node. All of the services can be started via docker compose (see: `./docker/docker-compose.base.yml`). The commands to start these services are:
 
 ```sh
-./stack chain-clean
-./stack chain
-# then in another terminal
-./stack chain-boot
+./stack compose-init # This will clean the chain and boot it
```

Comment:
When running `compose-init` a second time after running the base services, the following error is reported:

```
Error response from daemon: remove lilypad_chain-data: volume is in use - [440d450180a1295ee4d50c1f79e37f565d761cc59e9ff2de5f1f2b3e522c5685]
```

Everything runs fine starting the base services a second time without running `compose-init` a second time, but should this command give us a clean slate? (Not necessary on the PR, this does what we need for this PR.)